src/resources/css/Common.module.css

```diff
@@ -2,6 +2,12 @@ body {
   margin: 0;
 }
 
+html {
+  scrollbar-gutter: stable;
+  scrollbar-width: thin;
+  scrollbar-color: var(--ds-color-base-default) var(--ds-color-background-subtle);
```

Comment:
<img width="404" alt="Screenshot 2025-04-11 at 11 57 29" src="https://github.com/user-attachments/assets/51ff5cb9-f846-488d-98b2-31056a213904" />

I notice that adding background-color here, the scrollbar in the tabs goes a little crazy. Maybe setting the background color on the html elemt itself is a better solution? 
```suggestion
  scrollbar-color: var(--ds-color-base-default); 
  background-color: var(--ds-color-background-subtle);
```

Reply:
Good catch!
What do you mean with the html element itself? 🤔 
As far as I could see (though, I'm not really an expert on this,) the scrollbar for the entire page is not set by us, but belongs to the `html`-tag that encompasses our code, so the only place I was able to set a different color than the default, was to set it here.

Also, I don't see this artifact either when using the mobile simulator, only when making the desktop screen thin. Is it just me? 😅

Reply:
That being said, I'm not very happy with the approach of setting the color for the full page and then having to override it everywhere else, so I would love it if you had a suggestion of another alternative!